packages/web-components/tests/e2e-storybook/cypress/integration/card/card.e2e.js

```diff
@@ -7,6 +7,13 @@
 
 'use strict';
 
+import getCssPropertyForRule from '../../utils/get-css-property-for-rule';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import settings from 'carbon-components/es/globals/js/settings';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+const { prefix } = settings;
```

Comment:
These imports will fail in browserstack, hardcode the prefixes for now. I'm looking into setting up global variables in Cypress which we can use instead.